app/assets/stylesheets/pages/_login.scss

```diff
@@ -0,0 +1,9 @@
+.login-img{
+  height: 15rem;
+  background-image: asset-url('login.jpg');
+  background-size: cover;
+  background-position: center center;
+  @include media-breakpoint-up(md){
+    height: auto;
+  }
+}
```

Comment:
sort alphabetically this block, add spaces between the class names and curly braces, and add a line break in the `5` line

Reply:
Done, for the moment the form doesn't have any functionality, it's only design.